src/modules/Channel/components/SuggestedReplies/index.tsx

```diff
@@ -0,0 +1,43 @@
+import './index.scss';
+import React, { useState } from 'react';
+
+interface Props {
+  replyOptions: string[];
+  onSendMessage: ({ message }: { message: string }) => void;
+}
+
+const SuggestedReplies = ({ replyOptions, onSendMessage }: Props) => {
+  const [replied, setReplied] = useState<boolean>(false);
+
+  const onClickReply = (
+    event: React.MouseEvent<HTMLDivElement>,
+    option: string,
+  ) => {
+    event.preventDefault();
+    onSendMessage({ message: option });
+    setReplied(true);
```

Comment:
How do we know that this message has been responded to when re-entering the channel next time, once the `replied` state has been reset?

Reply:
Ah, I checked here `message.messageId === currentGroupChannel?.lastMessage.messageId`
never mind!